tests/common.sh

```diff
@@ -19,7 +19,6 @@
 set -eo pipefail
 
 TAG=${TAG:-dev}
-NODOCKER=${NODOCKER:-}
```

Comment:
can you also include the new 'sops' function in this PR?

Reply:
Done